jobserver/templates/index.html

```diff
@@ -56,11 +56,27 @@
     </div>
 
     {% if user_can_run_jobs %}
-    <div class="d-flex justify-content-center">
-      <a class="btn btn-lg btn-primary" href="{% url 'workspace-create' %}">
-        Or Add a New Workspace
-      </a>
+    <div class="text-center">
+      <p><strong>OR</strong></p>
+
+      <p>
+        Pick {% if request.user.orgs.count > 1 %}an Organisation then {% endif %}a Project to create a Workspace in
```

Comment:
Should Organisation/Project/Workspace be organisation/project/workspace?

Reply:
Yes 😂